fetcher/jvn/xml/jvn.go

```diff
@@ -64,6 +67,10 @@ type Cvss struct {
 	Version  string `xml:"version,attr"`
 }
 
+type CertLink struct {
```

Comment:
__[Golint]__ exported type CertLink should have comment or be unexported ([link](https://sider.review/gh/repos/54628930/pulls/129#issue-4304147))

<sub>posted by Sider</sub>